### PR TITLE
Add component IDs for impala JDBC Java plugin

### PR DESCRIPTION
**Pull request by lv-lifeng**
- [ ] If this pull request closes/resolves/fixes an existing issue, replace the issue number. Closes #<issue number>.
- [x] Update the [`CHANGES` log](https://github.com/apache/skywalking/blob/master/docs/en/changes/changes.md).

### CODE REVIEW

**Comment by wu-sheng** [comment]
Please update changelog.

File: docs/en/changes/changes.md [hunk 1]
```diff
@@ -5,9 +5,12 @@
 
 #### OAP Server
 
+Add 133 134 component ID for impala jdbc java plugin.
```
Comment (wu-sheng) [inline docs/en/changes/changes.md:8]:
```suggestion
* Add component ID(133) for impajdbcJDBCJava agent plugin and component ID(134) for impala server.
```

Please be accurate.

File: docs/en/changes/changes.md [hunk 2]
```diff
@@ -5,9 +5,12 @@
...
 
 #### UI
 
+Add impala icon for impala jdbc java plugin.
```
Comment (wu-sheng) [inline docs/en/changes/changes.md:12]:
```suggestion
```

I don't think you have added the icon to UI.

Comment (lv-lifeng) [reply docs/en/changes/changes.md]: ui repo, https://github.com/apache/skywalking-booster-ui/pull/153

Comment (wu-sheng) [reply docs/en/changes/changes.md]: Then we need to wait for that PR merged, and you update the UI submodule in this PR.

File: docs/en/changes/changes.md [hunk 3]
```diff
@@ -5,9 +5,12 @@
 
 #### OAP Server
 
+Add component ID(133) for impajdbcJDBCJava agent plugin and component ID(134) for impala server
 
 #### UI
 
+Add impala icon for impajdbcJDBCJava agent plugin.
```
Comment (wu-sheng) [inline docs/en/changes/changes.md:12]:
```suggestion
* Add impala icon for impala JDBC Java agent plugin.
```

File: docs/en/changes/changes.md (hunk 2 above)
Comment (lv-lifeng) [reply docs/en/changes/changes.md]: Okey.

Comment (wu-sheng) [reply docs/en/changes/changes.md]: UI PR is merged, please update the git submodule.

File: docs/en/changes/changes.md [hunk 4]
```diff
@@ -5,9 +5,12 @@
 
 #### OAP Server
 
+* Add component ID(133) for impala JDBC Java agent plugin and component ID(134) for impala server
 
 #### UI
 
+* Add impala icon for impala JDBC Java agent plugin.
```
Comment (wu-sheng) [inline docs/en/changes/changes.md:12]:
```suggestion
* Fix: tab active incorrectly, when click tab space 
* Add impala icon for impala JDBC Java agent plugin.
```

Comment (wu-sheng) [reply docs/en/changes/changes.md:12]: This is a changelog included in your submodule update.

**Review by wu-sheng — APPROVED** [review]

**Pull request merged**